outputStats/result.csv

```diff
@@ -0,0 +1,10 @@
+IoTDeviceName,Iteration,Count
```

Comment:
This result looks to be stale (IoTDeviceName -> DestinationVm)

Reply:
This file was used to keep output directory in the repo. I've changed it so the output directory is created if needed by the script itself